doc/start/documenting-ceph.rst

```diff
@@ -667,7 +667,7 @@ improves the readability of the document in a command line interface.
 
 
 .. _Python Sphinx: http://sphinx-doc.org
-.. _resturcturedText: http://docutils.sourceforge.net/rst.html
+.. _restructuredText: http://docutils.sourceforge.net/rst.html
```

Comment:
this probably broke the link. Note this is not a user-visible string, but an identifier.

Reply:
Ah indeed, there's no point fixing this so I've reverted this specific change.

That said, a `grep` search for `resturcturedText` doesn't show any other occurrence in this files or elsewhere. Also, none of the occurrences of `reStructuredText`in this page is a link:
https://docs.ceph.com/en/latest/start/documenting-ceph/